exo1/lib/screens/users/add_user_toolbar.dart

```diff
@@ -1,15 +1,40 @@
+import 'package:exo1/domain/user.dart';
+import 'package:exo1/domain/user_repository.dart';
 import 'package:flutter/material.dart';
 
-class AddUserToolbar extends StatelessWidget {
+class AddUserToolbar extends StatefulWidget {
+  @override
+  _AddUserToolbarState createState() => _AddUserToolbarState();
+}
+
+class _AddUserToolbarState extends State<AddUserToolbar> {
+  var newUser = '';
+  TextEditingController _controller = TextEditingController();
   @override
   Widget build(BuildContext context) {
     return Container(
       padding: EdgeInsets.all(8.0),
       child: Row(
         children: [
-          Flexible(child: TextField()),
+          Flexible(
+              child: TextField(
+            controller: _controller,
+            onChanged: (String value) {
+              setState(() {
+                newUser = value;
+              });
+            },
+          )),
```

Comment:
Add a comma between the 2 parenthesis to fix the indentation -> `),)`
When saving in VS Code or Android Studio, the auto-format should re-indent everything correctly

The formatter can be a bit tricky with that